src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2420,13 +2420,10 @@ def create_function(cmd, resource_group_name, name, storage_account, plan=None,
             else:
                 site_config.app_settings.append(NameValuePair(name='WEBSITES_ENABLE_APP_SERVICE_STORAGE',
                                                               value='true'))
-                if runtime not in RUNTIME_TO_IMAGE_FUNCTIONAPP[functions_version].keys():
+                if runtime not in RUNTIME_TO_SUPPORTED_VERSIONS_FUNCTIONAPP[functions_version]:
                     raise CLIError("An appropriate linux image for runtime:'{}' was not found".format(runtime))
```

Comment:
Include runtime version in the error message

Reply:
So if the user specifies a `runtime_version`, an error message will be emitted here: https://github.com/Azure/azure-cli/blob/dd691a3bff8ffa5230871ecb3ca7bbc216cdbf82/src/azure-cli/azure/cli/command_modules/appservice/custom.py#L2399-L2401

I added `functions_version` to this error message to make it more clear